treeherder/perf/models.py

```diff
@@ -155,7 +155,13 @@ class PerformanceDatum(models.Model):
 
     class Meta:
         db_table = 'performance_datum'
-        index_together = ('repository', 'signature', 'push_timestamp')
+        index_together = [
+            # this should speed up the typical "get a range of performance datums" query
+            ('repository', 'signature', 'push_timestamp'),
+            # this should speed up the compare view in treeherder (we only index on
+            # repository because we currently filter on it in the query)
+            ('repository', 'signature', 'push'),
+            ('repository', 'signature', 'job')]
         unique_together = ('repository', 'job', 'push', 'signature')
```

Comment:
Can you see a way to make the field query order match the `unique_together` order here? It's just that `('repository', 'job', 'push', 'signature')` overlaps with the others, were the field order adjusted.

Reply:
I actually didn't mean to add the ('repository', 'signature', 'job') one. Ok if I just take that out and leave this as-is?

Reply:
sgtm :-)